src/userCLI/src/main/kotlin/org/vaccineimpact/api/security/AddUserToGroup.kt

```diff
@@ -0,0 +1,71 @@
+package org.vaccineimpact.api.security
+
+import org.vaccineimpact.api.db.JooqContext
+import org.vaccineimpact.api.db.Tables.MODELLING_GROUP
+import org.vaccineimpact.api.models.Scope
+import org.vaccineimpact.api.models.permissions.ReifiedRole
+import kotlin.system.exitProcess
+
+sealed class GroupDefinition
+{
+    abstract fun getGroups(db: JooqContext): Iterable<String>
+
+    class GroupList(val groups: List<String>): GroupDefinition()
+    {
+        override fun getGroups(db: JooqContext) = groups
+    }
+    class AllGroups: GroupDefinition()
+    {
+        override fun getGroups(db: JooqContext): Iterable<String>
+        {
+            return db.dsl.select(MODELLING_GROUP.ID)
+                    .from(MODELLING_GROUP)
+                    .fetch()
+                    .map { it[MODELLING_GROUP.ID] }
+        }
+    }
+}
+
+class AddUserToGroupOptions(
+        val username: String,
+        val groups: GroupDefinition
```

Comment:
so here I would just make this a list of strings (group names), line 48 you call the 'getAllGroups' function, line 52 you just take the list of command line args